ReactSkia/components/RSkComponentImage.cpp

```diff
@@ -200,9 +200,11 @@ void RSkComponentImage::drawAndSubmit() {
 // callback for remoteImageData
 bool RSkComponentImage::processImageData(const char* path, char* response, int size) {
   decodedimageCacheData imageCacheData;
+  auto component = getComponentData();
+  auto const &imageProps = *std::static_pointer_cast<ImageProps const>(component.props);
   // Responce callback from network. Get image data, insert in Cache and call Onpaint
   sk_sp<SkImage> remoteImageData = RSkImageCacheManager::getImageCacheManagerInstance()->findImageDataInCache(path);
-  if(remoteImageData) {
+  if(remoteImageData && strcmp(path,imageProps.sources[0].uri.c_str()) == 0) {
```

Comment:
if remoteImageData is true and strcmp is false , we will end up doing strcmp operation twice in this if else logic .
so can we do this once and store in a variable and use it.